README.md

```diff
@@ -37,11 +37,11 @@ and are mounted.
                            Defaults to `vg`.
 `capra_lvm_filesystem_fstype`: Default file system type placed on the logical
                                volume.
-			       Defaults to `xfs`.
-`capra_lvm_filesystem_mount_parent`: Default path where the volumes will be
-				     mounted to.  The name of the volume will
-				     be appended.
-				     Defaults to `/mnt`.
+                               Defaults to `xfs`.
+`capra_lvm_filesystem_parent_path`: Default path where the volumes will be
+                                    mounted to.  The name of the volume will
```

Comment:
Not sure how the format of this list is supposed to work. It clearly doesn't. Use a plain itemized list:

```
* `foo:` bar with long lines
  when wrapped they
  are not indented
* `bar`: blubb
```